Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>Would be good to explain <em>why</em> you are using DBSCAN and PCA here. What does the reader/user get from these? Or (in the case of DBSCAN) how you use the results in later steps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>